Resources/Audio/Animals/attributions.yml

```diff
@@ -117,24 +117,42 @@
   license: "CC-BY-4.0"
   copyright: "Audio is recorded/created by Pfranzen 'FreeSound.org'. The original audio was trimmed and renamed"
   source: "https://freesound.org/people/pfranzen/sounds/322744/"
-  
+
 - files: ["dog_bark1.ogg"]
   license: "CC0-1.0"
   copyright: "Audio is recorded/created by KFerentchak 'FreeSound.org'. The original audio was trimmed and renamed"
-  source: "https://freesound.org/people/KFerentchak/sounds/235912/"  
-  
+  source: "https://freesound.org/people/KFerentchak/sounds/235912/"
+
 - files: ["dog_bark2.ogg"]
   license: "CC0-1.0"
   copyright: "Audio is recorded/created by KFerentchak 'FreeSound.org'. The original audio was trimmed and renamed"
-  source: "https://freesound.org/people/KFerentchak/sounds/235912/"  
-  
+  source: "https://freesound.org/people/KFerentchak/sounds/235912/"
+
 - files: ["dog_bark3.ogg"]
   license: "CC0-1.0"
   copyright: "Audio is recorded/created by KFerentchak 'FreeSound.org'. The original audio was trimmed and renamed"
   source: "https://freesound.org/people/KFerentchak/sounds/235912/"
-  
+
 - files: ["nymph_chirp.ogg"]
   license: "CC-BY-SA-3.0"
   copyright: "Taken from ParadiseSS13"
   source: "https://github.com/ParadiseSS13/Paradise/commit/a34f1054cef5a44a67fdac3b67b811137c6071dd"
-  
\ No newline at end of file
+
+- files:
+    - fox1.ogg
+    - fox2.ogg
+    - fox3.ogg
+    - fox4.ogg
+    - fox5.ogg
+    - fox6.ogg
+    - fox7.ogg
+    - fox8.ogg
+    - fox9.ogg
+    - fox10.ogg
+    - fox11.ogg
+    - fox12.ogg
+    - fox13.ogg
+    - fox14.ogg
+  copyright: "Created by fujiwaranao"
```

Comment:
You are a fox? (Copyright needs to be where you took the sounds from and what modifications you made etc. Source is the page you downloaded the sounds from (example freesound) check above)

Reply:
yes actually we made the fox noises ourselves with our own vocal cords hehe